vision/google/cloud/vision/geometry.py

```diff
@@ -87,6 +101,19 @@ def from_api_repr(cls, response):
         longitude = response['latLng']['longitude']
         return cls(latitude, longitude)
 
+    @classmethod
+    def from_pb(cls, response):
```

Comment:
Shouldn't the argument be named `location_info`? The fact that it came as a response is less relevant to what it is?